package.json

```diff
@@ -29,6 +29,7 @@
     "@types/react": "^16.9.32",
     "@uiw/codemirror-theme-sublime": "^4.21.7",
     "@uiw/react-codemirror": "^4.21.7",
+    "@uiw/react-color-sketch": "^2.1.0",
```

Comment:
remove https://github.com/fntechgit/summit-admin/blob/71f804fd66cebf6076fb95a62a2d3f6e32eb6436/package.json#L80 bc is not used anymore and regenerate the lock file